ssh-u2f.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
```

Comment:
Did you buy the blue u2f yubikeys or did you buy daplug?
The yubikey edge also work as a normal otp token.